src/auth/AxiosJwtAuthService.test.jsx

```diff
@@ -2,39 +2,24 @@
 import axios from 'axios';
```

Comment:
@abutterworth - Bang up job on these tests.  It's a testament to how well written they are that I barely had to change them to have them work with the new code shape.